apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx

```diff
@@ -194,6 +194,13 @@ export default class ModerationNotification extends Component {
     })
   }
 
+  translatedReportText (reportsFound) {
+    const tmp = django.ngettext('kosmo', '\'s {}comment{} has been reported 1 time since it\'s creation', '\'s {}comment{} has been reported %s times since it\'s creation', reportsFound)
```

Comment:
nice! maybe breaking `ngettext` parameters into new line for better readability?